//
// PROJECT:         Aspia
// FILE:            proto/desktop_session.proto
// LICENSE:         Mozilla Public License Version 2.0
// PROGRAMMERS:     Dmitry Chapyshev (dmitry@aspia.ru)
//

syntax = "proto3";

option optimize_for = LITE_RUNTIME;

import "desktop_session_message.proto";

package aspia.proto.desktop;

message HostToClient
{
    VideoPacket video_packet            = 1;
    CursorShape cursor_shape            = 2;
    ClipboardEvent clipboard_event      = 3;
    SessionConfigRequest config_request = 4;
}

message ClientToHost
{
    PointerEvent pointer_event     = 1;
    KeyEvent key_event             = 2;
    ClipboardEvent clipboard_event = 3;
    SessionConfig config           = 4;
}
